pyproject.toml

```diff
@@ -7,8 +7,10 @@ requires = [
     "Cython>=0.29.32,<3.0.0",
```

Comment:
Is 3.1.2 any better?

If so let's change this line to 3.1.2, and also update the comment above.

Reply:
@piskvorky I removed the upper version bound and tested with 3.1.2. However builds/tests take ages. It seems that Cython >=3 has stricter type checking, making it slower.

Reply:
I doubt it's type checking that takes ages. Maybe something (a dependency) gets built from source that used to be pulled pre-built before?

Reply:
@piskvorky I tried several things, unfortunately no luck. Especially Testing of test_doc2vec.py::TestDoc2VecModel seems to be much slower on Cython 3.1.2.
With Upgrading to 3.1.2 Python 3.13 seems to work however?!